### PR TITLE
GUT-179: fix padding and fonts

### PR DESCRIPTION
**Pull request by ethitter**
Revise `story-grid` module per designs.

### Doneness Checklist:

Pre-release # (or n/a):

- [x] Updated root CHANGELOG.md with summary of changes under `Unpublished` section
- [x] `npm run prod` in this repo outputs expected changes (excepting the issue with re-ordered partials in larva-css algorithms partials - see [LRVA-1885](https://jira.pmcdev.io/browse/LRVA-1885))
- [ ] If changes to build scripts or the Node.js server, tested changes in pmc-spark [via a pre-release](https://confluence.pmcdev.io/x/XhOeAw)
- - [ ] If changes to build tools: npm scripts `prod`, `lint`, and `dev` scripts run as expected
- - [ ] If changes to Larva server: Static site generates as expected in a theme  (avail. on a URL {brand}.stg.larva.pmcdev.io)

### CODE REVIEW

**Comment by ethitter** [comment]
@laras126 I'm a bit stuck on this and need your guidance. 

1. I don't think I should be using the `lrv-u-font-family-*` classes anymore, but I'm not sure how to apply the correct fonts otherwise.
1. There's a problem with the alignment of the byline and the "By " text that precedes it, which I haven't been able to resolve: https://pmc-larva-git-feature-gut-179-penske-media-corp.vercel.app/larva/modules/story-grid/

**Comment by laras126** [comment]
@ethitter 

1. The design should have some documentation about which font family to use, then choose that one or the closest looking one from here - https://pmc-larva-git-feature-gut-179-penske-media-corp.vercel.app/larva/style-guide

We can't see the indiewire design tokens on this view, but if you start the Larva server locally from the root of this repo and add a query string ?tokens=indiewire you can preview them.

2. Add `lrv-u-margin-tb-00 lrv-u-padding-l-050` to the c_tagline_classes in the byline and you can use `lrv-u-align-items-center` on the `li` to vertically align them center.

**Comment by ethitter** [comment]
@laras126 Thanks for the help, that resolved all of my (current) issues! This is ready for another look

File: packages/larva-patterns/modules/story-grid/story-grid.prototype.js [hunk 1]
```diff
@@ -2,6 +2,15 @@ const clonedeep = require( 'lodash.clonedeep' );
 
 const story_card = clonedeep( require( '../story/story.vertical' ) );
 
+story_card.c_dek.c_dek_classes = 'lrv-a-font-body-s lrv-a-hidden@mobile-max lrv-u-margin-a-00';
```
Comment (laras126) [inline packages/larva-patterns/modules/story-grid/story-grid.prototype.js:5]: Perfect!! <3

**Review by laras126 — APPROVED** [review]
Lovely!

**Pull request merged**